app/stac_api/management/commands/remove_expired_items.py

```diff
@@ -0,0 +1,68 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management.base import CommandParser
+from django.utils import timezone
+
+from stac_api.models import Asset
+from stac_api.models import Item
+from stac_api.utils import CommandHandler
+from stac_api.utils import CustomBaseCommand
+
+
+class Handler(CommandHandler):
+
+    def delete(self, instance, object_type):
```

Comment:
Looking at the code below, you're actually not passing an `instance` here, but rather a set of instances. (Actually, from a type POV I believe it's a queryset).
Update: OK on the second call it is *just* an instance :)